src/hnsw.c

```diff
@@ -7,6 +7,7 @@
 #include <catalog/namespace.h>
 #include <commands/vacuum.h>
 #include <float.h>
+#include <hnsw/utils.h>
```

Comment:
is this our header, included a second time?

Reply:
Removed duplicate include